src/services/functionAppService.test.ts

```diff
@@ -36,18 +37,16 @@ describe("Function App Service", () => {
   const getFunctionResponse = "hello function";
 
   const baseUrl = "https://management.azure.com"
-  const masterKeyUrl = `https://${app.defaultHostName}/admin/host/systemkeys/_master`;
+  const masterKeyUrl = "https://management.azure.comappId/host/default/listkeys?api-version=2016-08-01";
```

Comment:
Looks like this key path finally went into actual API?  I recall seeing some weird issue where API keys were actually not working correctly.  Can you confirm that is working as expected from APIM -> Function app.

Reply:
I'll look into this. I'll probably just revert that change.

Reply:
Fixed this.